app/Resources/views/home/index.html.twig

```diff
@@ -42,6 +42,19 @@
             </a>
         </div>
 
+        <div class="content">
```

Comment:
Can you put this block around `{% if enable_canary %}...{% endif %}` so we can disable it in prod while it's not ready but still merge this PR?